docs/blog/newsletter-april-2024.rst

```diff
@@ -0,0 +1,143 @@
+
+Some respondents were adamant that it doesn't help them. Some because their company's security policies forbid its use. Others found it useful for generic content, where it can act like a focused web search but without any ads, but not for specialized questions about a product. Others said it wouldn't help them because writing the first draft is how they learn about the product.
+
+Some of the people who didn't find it personally helpful thought it might help non-writers or non-native speakers. People who struggle getting words on the page might benefit from the ability of large-language models (LLMs) to put together sentences in a commonly understood order.
+
+Others were adamant that LLMs have really helped them speed up their documentation process. For security, it's important to choose a service that doesn't share prompts and responses with the LLM providers for training. 
+
+Even in the best of circumstances, LLMs were compared to "extremely knowledgeable but inexperienced interns." They require dedicated focus and crafting of prompts and other tools to get them to do what you want. It can help to use `retrieval-augmented generation <https://stackoverflow.blog/2023/10/18/retrieval-augmented-generation-keeping-llms-relevant-and-current/>`__ to keep the LLM focused.
+
+The discourse on AI's impact on documentation creation encompasses ethical, societal, and professional dimensions. By fostering ongoing dialogue and collaboration within the technical community, organizations can navigate the complexities of AI integration while promoting responsible and inclusive documentation practices.
+
+----------------------------------
+Questions To Ask as an Interviewer
+----------------------------------
+
+The WTD community offers valuable resources for job seekers, many captured in the `Guide to Hiring and Getting Hired </hiring-guide/>`__ and the `Getting Hired </topics/#getting-hired>`__ topics. This month in Slack, the focus shifted to the other side of the interview, as documentarians and managers listed their favorite questions to ask candidates. Here's a roundup of the questions they shared, along with any reasons they mentioned.
+
+- How did you get into technical writing?
+
+ - This is an easy starter question that gets candidates comfortable, gauges their enthusiasm for the job, and tells you about their career path.
+
+- What do you do outside your regular role to advocate for your profession?
+
+- Do you take the initiative on docs retros or create presentations for the team?
+
+- Do you like to stretch beyond writing?
+
+- What other profession would you like to try?
+
+ - This isn't a common interview question, so it shows how candidates respond to something they may not be prepared for.
```

Comment:
This is a little too conditional for my tastes. Especially given the assumption.